cve_bin_tool/checkers/mysql.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for mysql CLI
+References:
+https://www.mysql.com/
+https://www.cvedetails.com/vulnerability-list/vendor_id-93/product_id-21801/Oracle-Mysql.html
+"""
+from . import Checker
+
+
+class MysqlChecker(Checker):
+
+    CONTAINS_PATTERNS = [
+        r"For information about MySQL products and services, visit:",
+        r"Oracle and/or its affiliates. All rights reserved.",
```

Comment:
I think this pattern is probably going to match a bunch of other things provided by Oracle (e.g. Java, berkelydb) so it's probably going to generate a lot of false postiives.